src/config/keys.ts

```diff
@@ -279,6 +279,28 @@ export const CURRENT_MEMBER_STORAGE_KEY = [
 
 export const OWN_PUBLIC_PROFILE_KEY = ['own-profile'];
 export const buildPublicProfileKey = (memberId: UUID) => ['profile', memberId];
+
+export const itemsWithGeolocationKeys = {
+  allBounds: [ITEMS_CONTEXT, 'map'],
+  inBounds: ({
+    lat1,
+    lat2,
+    lng1,
+    lng2,
+  }: {
+    lat1: number;
+    lat2: number;
+    lng1: number;
+    lng2: number;
+  }) => [...itemsWithGeolocationKeys.allBounds, { lat1, lat2, lng1, lng2 }],
+};
+
+export const buildItemGeolocationKey = (itemId?: UUID) => [
```

Comment:
I think we should really create a key factory for item keys so they are ordered hierarchically.